dashboard/app/assets/javascripts/components/report_abuse_form.jsx

```diff
@@ -18,16 +18,24 @@ window.dashboard.ReportAbuseForm = (function (React) {
    */
   var AgeDropdown = React.createClass({
     propTypes: {
-      age: React.PropTypes.string
+      age: React.PropTypes.oneOfType([
+        React.PropTypes.string,
+        React.PropTypes.number
+      ])
     },
 
     render: function () {
       var style = $.extend({}, {width: DROPDOWN_WIDTH}, this.props.style);
 
+      var age = this.props.age;
+      if (typeof(age) === 'number') {
+        age = age.toString();
```

Comment:
Would it be simpler to just always call `.toString()` since it will be fine in both cases?